src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/DiagramScene.java

```diff
@@ -217,7 +217,7 @@ public void filteredChanged(SelectionCoordinator coordinator) {
                         selectedFigures.add(figure);
                     }
```

Comment:
```suggestion
                    }
                    for (Slot slot : figure.getSlots()) {
                        if (!Collections.disjoint(slot.getSource().getSourceNodesAsSet(), ids)) {
                            highlightedObjects.add(slot);
                        }
                    }
```
I am not sure what your intent was in adding the slots to the selected objects. If you wanted the slots to be selected globally in "link global node selection" mode, you need to add the following code to make it work

Reply:
Even if this was not you intention, I think selecting the slots globally is a useful feature.

Reply:
Thanks for the suggestion! I added support for this now, however after considering the amount of work required to implement proper interaction with slots in a consistent manner, I lean towards excluding the partial implementation from this changeset and leaving it instead for future work, see my reply to @chhagedorn.

Reply:
ok, I am fine with leaving this for future work